projects/packages/forms/src/dashboard/about/index.tsx

```diff
@@ -199,6 +200,24 @@ const About = () => {
 					<Details summary={ __( 'Is there a form responses limit?', 'jetpack-forms' ) }>
 						{ __( 'No.', 'jetpack-forms' ) }
 					</Details>
+					{ ! isSimple && (
+						<Details
+							summary={ __( 'Can I change Forms functionality via code?', 'jetpack-forms' ) }
+						>
+							{ createInterpolateElement(
+								__(
+									'Yes! You can change several aspects of Forms by using <filtersDocs>WordPress filters</filtersDocs>. Please see our <devDocs>developer documentation</devDocs> for full list of available filters and details.',
+									'jetpack-forms'
+								),
+								{
+									filtersDocs: (
+										<ExternalLink href="https://developer.wordpress.org/plugins/hooks/filters/" />
+									),
+									devDocs: <ExternalLink href={ getRedirectUrl( 'jetpack-contact-support' ) } />,
```

Comment:
TODO: now links to docs, but needs creating a new redirect for https://developer.jetpack.com/module/contact-form/ and using it here.